shell/browser/ui/cocoa/electron_menu_controller.mm

```diff
@@ -363,6 +369,13 @@ - (NSMenuItem*)makeMenuItemForIndex:(NSInteger)index
     NSMenu* submenu = MenuHasVisibleItems(submenuModel)
                           ? [self menuFromModel:submenuModel]
                           : MakeEmptySubmenu();
+
+    if (@available(macOS 14, *)) {
```

Comment:
It's probably worth documenting that the `palette` role is only available in macOS 14+

Reply:
[`cecf5f5`](https://github.com/electron/electron/pull/45538/commits/cecf5f565604b466892b5ff49ff4d385bab8f1e5)